packages/datadog_flutter_plugin/android/src/test/kotlin/com/datadoghq/flutter/DatadogLogsPluginTest.kt

```diff
@@ -215,6 +215,13 @@ class DatadogLogsPluginTest {
     }
 
     private val contracts = listOf(
+        Contract("addGlobalAttribute", mapOf(
+            "key" to ContractParameter.Type(SupportedContractType.STRING),
+            "value" to ContractParameter.Type(SupportedContractType.MAP),
```

Comment:
should it be `Any` here instead of `MAP`?

Reply:
The contracts don't really have a concept of "Any" so I give it the most complicated type I have.

At some point I may want to go back and add "Any" as a supported type (this is true in iOS as well)